language-server/src/test/kotlin/tools/samt/ls/SamtWorkspaceTest.kt

```diff
@@ -138,9 +138,9 @@ class SamtWorkspaceTest {
         workspace.addFolder(SamtFolder("file:///tmp/test".toPathUri()))
         val file1 = parseFile(SourceFile ("file:///tmp/test/foo.samt".toPathUri(), "package foo.bar record Foo {}"))
         workspace.setFile(file1)
-        assertNull(workspace.getRootPackage(file1.path))
+        assertNull(workspace.getSemanticModel(file1.path))
         workspace.buildSemanticModel()
-        val rootPackage = workspace.getRootPackage(file1.path)
+        val rootPackage = workspace.getSemanticModel(file1.path)
         assertNotNull(rootPackage)
```

Comment:
Nitpick: rename variable
```suggestion
        val semanticModel = workspace.getSemanticModel(file1.path)
        assertNotNull(semanticModel)
```